plan/2.0 Expand Training.md

```diff
@@ -38,338 +16,519 @@
-
-  - [ ] **M1.1**: Python
-    - [ ] **M1.1.1**: Create **[180 Labs]** new Python labs
-    - [ ] **M1.1.2**: Corralate new Python 'Labs' to 'Concepts'
-  - [ ] **M1.2**: Java + [ ] **M1.2.1**: Create **[180 Labs]** new Java labs + [ ] **M1.2.2**: Corralate new Python 'Labs' to 'Concepts'
-  - [ ] **M1.3**: Nodejs
-    - [ ] **M1.3.1**: Create **[180 Labs]** new NodeJS labs
-      - [ ] **M1.3.2**: Corralate new Python 'Labs' to 'Concepts'
-
-- [ ] **M2**: Create job descriptions for contractors for M3
-- [ ] **M3**: Extending current 'Lab' offerings to include 5 other langauges for the currently indexed standards (e.g., ASVS & WSTG) in the Security Knowledge Graph (SKG)
-  - [ ] **M3.1**: Ruby
-    - [ ] **M3.1.1**: Create **[280 Labs]** new Ruby labs
-    - [ ] **M3.1.1**: Corralate new Ruby 'Labs' to 'Concepts'
-  - [ ] **M3.2**: Rust
-    - [ ] **M3.3.1**: Create **[280 Labs]** new Rust labs
-    - [ ] **M3.2.2**: Corralate new Rust 'Labs' to 'Concepts'
-  - [ ] **M3.3**: Go
-    - [ ] **M3.3.1**: Create **[280 Labs]** new Go labs
-    - [ ] **M3.3.2**: Corralate new Go 'Labs' to 'Concepts'
-  - [ ] **M3.4**: DotNet/C#
-    - [ ] **M3.4.1**: Create **[280 Labs]** new DotNet/C# labs
-    - [ ] **M3.4.2**: Corralate new DotNet/C# 'Labs' to 'Concepts'
-  - [ ] **M3.5**: C/C++
-    - [ ] **M3.5.1**: Create **[280 Labs]** new C/C++ labs
-    - [ ] **M3.5.2**: Corralate new C/C++ 'Labs' to 'Concepts'
+- **M1**: Improving the current 'Lab' offering to cover the currently indexed standards (e.g., ASVS & WSTG) in the Security Knowledge Graph (SKG)
 
-### Budget Requests
+  - Python
+    - [ ] Create **[180 Labs]** new Python labs
+    - [ ] correlate new Python 'Labs' to 'Concepts'
+  - Java
+    - [ ] Create **[180 Labs]** new Java labs
+    - [ ] correlate new Java 'Labs' to 'Concepts'
+  - Nodejs
+    - [ ] Create **[180 Labs]** new NodeJS labs
+    - [ ] correlate new NodeJS 'Labs' to 'Concepts'
 
-#### Time & Resource Estimate
+- **M2**: Create job descriptions for contractors for M3
+
+  - [ ] Create a job description for secdev contributor per program language
+  - [ ] Get feedback on the job description
+  - [ ] post opening and interview candidates
+  - [ ] onboard secdev contributors
+
+- **M3**: Extending current 'Lab' offerings to include 5 other languages for the currently indexed standards (e.g., ASVS & WSTG) in the Security Knowledge Graph (SKG)
+
+  - Ruby
+    - [ ] Create **[280 Labs]** new Ruby labs
+    - [ ] Correlate new Ruby 'Labs' to 'Concepts'
+  - Rust
+    - [ ] Create **[280 Labs]** new Rust labs
+    - [ ] Correlate new Rust 'Labs' to 'Concepts'
+  - Go
+    - [ ] Create **[280 Labs]** new Go labs
+    - [ ] Correlate new Go 'Labs' to 'Concepts'
+  - DotNet/C#
+    - [ ] Create **[280 Labs]** new DotNet/C# labs
+    - [ ] Correlate new DotNet/C# 'Labs' to 'Concepts'
+  - C/C++
+    - [ ] Create **[280 Labs]** new C/C++ labs
+    - [ ] Correlate new C/C++ 'Labs' to 'Concepts'
```

Comment:
Are these based on existing first and second tier languages (existing popular and emerging popular)? Recommend potentially adding a milestone for creating criteria to add additional languages against popularity of use. This would allow us to develop content for existing languages as well as ones becoming popularized so we effectively burn the candle from both ends.